### DISCUSSION TITLE
Wix toolset for Windows for ARM64

### DISCUSSION THREAD

**Discussion by i-lina** [wixtoolset/issues#8050] [WiX Development]
Hello,
Windows on ARM64 is an up and coming architecture with lots of new development and tools and libraries. Currently v4.0 allows installation using dot net for Windows ARM64. It would be nice to get an installer support for Windows ARM64 architecture.
Could you please have the next release support Windows ARM64 as a target?

**Comment by barnson** [comment]
WiX v3.14, v4.0, and v5-rc all support Arm64.

**Comment by robmen** [comment]
WiX Toolset v4 already supports ARM64. What did you find was missing?

**Comment by i-lina** [comment]
Thanks @robmen. Is there an installer available?

**Reply by robmen** [reply]
You should read the Getting Started documentation. That explains how to get WiX v4.
